specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/examples/BackupListByServer.json

```diff
@@ -0,0 +1,126 @@
+            "name": "daily_20210619T160533",
+            "type": "Microsoft.DBforPostgreSQL/flexibleServers/backups"
+          },
+          {
+            "properties": {
+              "backupType": "FULL",
+              "source": "Automatic",
+              "completedTime": "2021-06-20T16:05:41.9200138+00:00"
+            },
+            "id": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.DBforPostgreSQL/flexibleServers/postgresqltestserver/backups/daily_20210620T160538",
+            "name": "daily_20210620T160538",
+            "type": "Microsoft.DBforPostgreSQL/flexibleServers/backups"
+          },
+          {
+            "properties": {
+              "backupType": "FULL",
+              "source": "Automatic",
+              "completedTime": "2021-06-21T16:05:48.8528447+00:00"
+            },
+            "id": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.DBforPostgreSQL/flexibleServers/postgresqltestserver/backups/daily_20210621T160543",
+            "name": "daily_20210621T160543",
+            "type": "Microsoft.DBforPostgreSQL/flexibleServers/backups"
+          },
+          {
+            "properties": {
+              "backupType": "FULL",
+              "source": "Automatic",
+              "completedTime": "2021-06-22T16:08:06.3121688+00:00"
+            },
+            "id": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.DBforPostgreSQL/flexibleServers/postgresqltestserver/backups/daily_20210622T160803",
+            "name": "daily_20210622T160803",
+            "type": "Microsoft.DBforPostgreSQL/flexibleServers/backups"
+          },
+          {
+            "properties": {
+              "backupType": "FULL",
+              "source": "Automatic",
+              "completedTime": "2021-06-22T21:08:10.5057354+00:00"
+            },
+            "id": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.DBforPostgreSQL/flexibleServers/postgresqltestserver/backups/daily_20210622T210807",
+            "name": "daily_20210622T210807",
+            "type": "Microsoft.DBforPostgreSQL/flexibleServers/backups"
+          },
+          {
+            "properties": {
+              "backupType": "FULL",
+              "source": "Automatic",
+              "completedTime": "2021-06-23T21:24:16.9401531+00:00"
+            },
+            "id": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.DBforPostgreSQL/flexibleServers/postgresqltestserver/backups/daily_20210623T212413",
+            "name": "daily_20210623T212413",
+            "type": "Microsoft.DBforPostgreSQL/flexibleServers/backups"
+          },
+          {
+            "properties": {
+              "backupType": "FULL",
```

Comment:
>FULL [](http://example.com/codeflow?start=29&length=4)

Nit, did it have to be all caps?

Reply:
This looks like an enumeration value, but I don't see the enumeration value defined anywhere in your OpenApiSpec yet. Was it defined elsehwere?

Reply:
No it doesn't have to be FULL in caps. Its of type string and api doesnt define this as an enum

Reply:
I assume you treat it as an enum with an explicit set of acceptable values though when you're processing the request?